sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/XmlExternalEntityShouldNotBeParsed.cs

```diff
@@ -173,15 +174,15 @@ public static (CSharpObjectInitializationTracker xmlDocumentTracker, CSharpObjec
                     isAllowedConstantValue: constantValue => false,
                     trackedTypes: XmlDocumentTrackedTypes,
                     isTrackedPropertyName: propertyName => "XmlResolver" == propertyName,
-                    isAllowedObject: IsAllowedObject,
+                    isAllowedObject: (symbol, _, __) => IsAllowedObject(symbol),
```

Comment:
is it me or are there two underscores `__` ?
is it intentional?
(also see line 185)

Reply:
Indeed there are two underscores and it was intentional. AFAICT it's not possible to use _ for both parameters. The compiler will complain that the parameter name is a duplicate.